exporter/collector/metricsexporter.go

```diff
@@ -878,16 +888,31 @@ func (m *metricMapper) labelDescriptors(pm pdata.Metric) []*label.LabelDescripto
 }
 
 // Returns (sum, count, percentile) metric names for a summary metric.
-func summaryMetricNames(name string) (string, string, string) {
+func (m *metricMapper) summaryMetricNames(name string) (string, string, string) {
```

Comment:
Nit: I worry that returning three strings from such a function makes it easy to mix up the order. I'd prefer a small struct even though it's a bit more verbose.

(I see it was already this way before the current PR, so feel free to fix this in a separate PR if you agree with the feedback.)

Reply:
added a TODO